config/packages/db_tools.yaml

```diff
@@ -28,6 +28,12 @@ db_tools:
     # (Use relative date/time formats : https://www.php.net/manual/en/datetime.formats.relative.php)
     #backup_expiration_age: '6 months ago' # default '3 months ago'
 
+    # Timeout for backups.
+    # backup_timeout: 1200 # default 600
```

Comment:
I would love to see this accepting date string such as "3 minutes" for example, this can be implemented by validating it using `\DateInterval::createFromDateString()` when not `\ctype_digit()` in the extension class.